src/plugins/intel_cpu/tests/functional/single_layer_tests/reduce_ops.cpp

```diff
@@ -303,6 +303,11 @@ std::vector<std::vector<ov::test::InputShape>> inputShapes_SmallChannel = {
     {{{{1, 5}, 3, {1, 5}, {1, 10}}, {{2, 3, 2, 2}, {2, 3, 2, 9}}}},
 };
 
+std::vector<std::vector<ov::test::InputShape>> inputShapes_SingleBatch = {
+    {{{}, {{1, 19, 2, 9}}}},
+    {{{{1, 5}, 19, {1, 5}, {1, 10}}, {{1, 19, 2, 2}, {1, 19, 2, 9}}}},
+};
```

Comment:
1. I am not sure how many threads is used in single layer test, but some machine do not have 19 cores in total, that never trigger the code path. At least we can reduce Channel size(19) to cover code path with higher possibility.
2. use 5D shape, and set OD not to 1, to test?

Reply:
and why not put these test shapes in inputShapes_5D in line 286, to avoid duplication.